git_machete/cli.py

```diff
@@ -364,6 +365,9 @@ def add_code_hosting_parser(command: str, subcommand_suffix: str, include_sync:
     status_parser.add_argument('-l', '--list-commits', action='store_true')
     status_parser.add_argument('-L', '--list-commits-with-hashes', action='store_true')
     status_parser.add_argument('--no-detect-squash-merges', action='store_true')
+    status_parser.add_argument('--squash-merge-detection',
```

Comment:
The tests are now failing... but pls refrain from fixing them, maybe once you migrate to `git log --patch` or `git cherry`, the extra option/git config key won't even be necessary and what's now `exact` mode will be made the default 🤔